modules/core/src/test/scala/gem/LocationSpec.scala

```diff
@@ -70,7 +70,7 @@ class LocationSpec extends FlatSpec with Matchers with PropertyChecks with Arbit
     }
   }
 
-  it should "produce a sorted list of Location.Middle" in {
+  ignore should "produce a sorted list of Location.Middle" in {
```

Comment:
I suspect the `foldRight` in `toBase10`.  Switching to `reverse` first and `foldLeft` may work, I guess.  Even if it does work, there may be better solutions.

```scala
loc.positions.take(len).reverse.foldLeft((Zero, One)) { case ((acc, pow), i) =>
```